README.md

```diff
@@ -37,7 +37,10 @@ You can find oneDPL samples at the [oneDPL Samples](https://github.com/oneapi-sr
 ## Support and Contribution
 Please report issues and suggestions via [GitHub issues](https://github.com/oneapi-src/oneDPL/issues).
 
+You can also contact oneDPL developers via [UXL Foundation Slack](https://slack-invite.uxlfoundation.org/) using
```

Comment:
To satisfy https://github.com/uxlfoundation/open-source-working-group/issues/6 , do we need to name the specific people who are monitoring the channel here in this PR, or is it enough to have assigned this?  It seems we may need to name the people here.

Reply:
Ideally there is a table similar to the one here listing resposible maintainers for different parts of the project. https://github.com/oneapi-src/oneDNN/blob/main/MAINTAINERS.md

You don't need to say who is monitoring the Slack specifically so long as someone is checking.